feconf.py

```diff
@@ -711,6 +712,10 @@ def get_empty_ratings():
 COMMIT_MESSAGE_SUBTOPIC_PAGE_DELETED = 'Subtopic page deleted.'
 COMMIT_MESSAGE_TOPIC_DELETED = 'Topic deleted.'
 
+QUESTION_STATUS_APPROVED = 'approved'
```

Comment:
Why do we have these when the review system is not implemented yet?

Reply:
Removed